GOVERNANCE-elections.md

```diff
@@ -0,0 +1,279 @@
+between one and three Election Officer(s) to administer the election. Elections
+Officers will be community members in good standing who are eligible to
+vote, are not running for Steering in that election, who are not currently part
+of the Steering Committee and can make a public promise of impartiality. They
+will be responsible for:
+
+- Making all announcements associated with the election
+- Preparing and distributing electronic ballots
+- Judging exception requests
+- Assisting candidates in preparing and sharing statements
+- Tallying voting results according to the rules in this charter
+
+### Eligibility to Vote
+
+Anyone who has at least [TODO:Number] contributions in the last 12 months is
+eligible to vote in the Steering election. Contributions are defined as opening PRs,
+reviewing and commenting on PRs, opening and commenting on issues, writing
+design docs, commenting on design docs, helping people on community forums, participating
+in working groups, and other efforts that help advance the project.
+
+[This dashboard](TODO: Devstats link)
+shows only GitHub based contributions and does not capture all the contributions
+we value. We expect this metric not to capture everyone who should be eligible
+to vote. If a community member has had significant contributions over the past
+year but is not captured in Devstats,
+they will be able to submit an exception form to the Elections Officer(s) who
+will then review and determine whether this member should be eligible to vote.
+All exceptions, and the reasons for them, will be recorded and saved by the
+officers.
+
+The electoral roll of all eligible voters will be captured at [TODO: Github election
+location] and the voters’ guide will
+be captured at [TODO: Github Election info location],
+similar to the
+[Kubernetes election process](https://github.com/kubernetes/steering/blob/master/elections.md).
+
+We are committed to an inclusive process and will adapt future eligibility
+requirements based on community feedback.
+
+### Candidate Eligibility
+
+Community members must be eligible to vote in order to stand for election (this
+includes voters who qualify for an exception). Candidates may self-nominate or
+be nominated by another eligible member. There are no term limits for Steering
+Committee members. Nothing prevents a qualified member from serving on both Steering
+and other groups simultaneously.
+
+To run for a seat, a candidate must additionally be at least a
+project Member as defined in
+[ROLES.md](TODO: Link to your contributor ladder).
+
+### Voting Procedure
+
+Elections will be held using a time-limited
+[Condorcet](https://en.wikipedia.org/wiki/Condorcet_method) ranking on
+using the IRV method. The top vote-getters will be elected to the open seats,
+with the exceptions for company representation discussed below.
+
+### Limitations on Company Representation
+
+No more than [TODO: Number] seats may be held by employees of the same organization (or
```

Comment:
From our earlier discussion, we were thinking no more than 2. We wanted to prevent any one company from having a simple majority.